providers/gce/gce_annotations.go

```diff
@@ -51,6 +51,9 @@ const (
 	// ALPHA feature - this may be removed in a future release.
 	ServiceAnnotationILBBackendShare = "alpha.cloud.google.com/load-balancer-backend-share"
 
+	// ServiceAnnotationLBIPv4 aim to instead of the deprecated spec field LoadBalancerIP from v1.24
+	ServiceAnnotationLBIPv4 = "service.beta.kubernetes.io/gcp-load-balancer-eip"
```

Comment:
Clarify that this annotation takes precedence over the field?

Reply:
@jpbetz  The corresponding description has been added